tests/src/commands/create-project.spec.ts

```diff
@@ -0,0 +1,58 @@
+import { handler as createProject } from "../../../src/commands/create-project";
```

Comment:
For JavaScript/TypeScript tests, they typically live alongside the actual code. Should we do the same here instead of splitting into a `tests` directory where we'll need to maintain similar structure code?

Example:
```
- src
  - commands
    create-project.ts
    create-project.spec.ts
    deposit.ts
    deposit.spec.ts
    ...
  - test-utils
    jestGlobalSetup.ts
    requestInterceptor.ts
    matchers.ts
    mockers.ts
    mocks.ts
```

Reply:
As far as I know there isn't single right way to do it. From my experience when there are a lot of actual scripts files (eg utils folder) having `.spec` file in the same folder makes it a little harder to quickly find what you are looking for. On the other hand having it in a separate folder it makes the file structure cleaner and separates the actual "source code" (src) from tests.

Reply:
Hm, actually I think I'll move tests file inside src indeed cause otherwise there is a lot of playing with tsconfig because of `rootDir` :)